spool_2/regex_spec.rb

```diff
@@ -4,73 +4,74 @@
 
   let(:phrase) { "It was a bright cold day in April, and the clocks were striking thirteen." }
 
-  it "matches using =~" do
+  it "matches using match" do
     expect(phrase).to __ /bright/
   end
+
   it "matches using .match" do
-    expect(/April/.__(phrase)).to be_true
+    expect(/April/.__(phrase)).to be_truthy
   end
 
-  # What's the difference between using =~ and .match?
+  # What's the difference between using match and .match?
```

Comment:
The problem with using find and replace I guess. I think this comment isn't relevant now.